<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tela_editar_produto</class>
 <widget class="QDialog" name="tela_editar_produto">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>826</width>
    <height>275</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>826</width>
    <height>275</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>826</width>
    <height>275</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Editar Produto</string>
  </property>
  <widget class="QGroupBox" name="gb_dados_produto">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>5</y>
     <width>816</width>
     <height>265</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>Dados do produto</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <widget class="QLabel" name="lb_nome">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>35</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="text">
     <string>Nome:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_nome">
    <property name="geometry">
     <rect>
      <x>175</x>
      <y>20</y>
      <width>451</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>60</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_fabricante">
    <property name="geometry">
     <rect>
      <x>68</x>
      <y>50</y>
      <width>347</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>60</number>
    </property>
   </widget>
   <widget class="QLabel" name="lb_fabricante">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>50</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="text">
     <string>Fabricante:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_codigo_barras">
    <property name="geometry">
     <rect>
      <x>422</x>
      <y>50</y>
      <width>99</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="text">
     <string>Código de Barras:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_codigo_barras">
    <property name="geometry">
     <rect>
      <x>521</x>
      <y>50</y>
      <width>105</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="inputMask">
     <string>0 000000 000000; </string>
    </property>
    <property name="text">
     <string>  </string>
    </property>
    <property name="maxLength">
     <number>15</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_quantidade">
    <property name="geometry">
     <rect>
      <x>72</x>
      <y>80</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="inputMask">
     <string>0000000000; </string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLabel" name="lb_quantidade">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>80</y>
      <width>67</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="text">
     <string>Quantidade:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="te_des_utilizacao">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>130</y>
      <width>621</width>
      <height>130</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Calibri'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_comentario">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>110</y>
      <width>133</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="text">
     <string>Descrição da utilização:</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="gv_imagem_produto">
    <property name="geometry">
     <rect>
      <x>631</x>
      <y>20</y>
      <width>180</width>
      <height>180</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>150</width>
      <height>150</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>200</width>
      <height>200</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_cancelar">
    <property name="geometry">
     <rect>
      <x>721</x>
      <y>236</y>
      <width>90</width>
      <height>24</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>24</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>90</width>
      <height>24</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Cancelar</string>
    </property>
    <property name="icon">
     <iconset resource="imagens.qrc">
      <normaloff>:/img/img/1345902747_button_cancel.png</normaloff>:/img/img/1345902747_button_cancel.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>18</width>
      <height>18</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_confirmar">
    <property name="geometry">
     <rect>
      <x>631</x>
      <y>236</y>
      <width>90</width>
      <height>24</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>24</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>90</width>
      <height>24</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Confirmar</string>
    </property>
    <property name="icon">
     <iconset resource="imagens.qrc">
      <normaloff>:/img/img/1345902916_camera_test.png</normaloff>:/img/img/1345902916_camera_test.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="lb_tipo">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>20</y>
      <width>28</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="text">
     <string>Tipo:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cb_tipo">
    <property name="geometry">
     <rect>
      <x>33</x>
      <y>20</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
      <kerning>false</kerning>
     </font>
    </property>
    <item>
     <property name="text">
      <string>Acessório</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Cosmético</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Decoração</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Estimulante</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Fetish</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Gel</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Higiene</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Jogo</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Kit</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Lubrificante</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Perfumaria</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Preservativo</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Prótese</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Vestuário</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Vibrador</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="btn_adicionar_imagem">
    <property name="geometry">
     <rect>
      <x>666</x>
      <y>205</y>
      <width>110</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Adicionar imagem</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_valorvenda">
    <property name="geometry">
     <rect>
      <x>352</x>
      <y>80</y>
      <width>88</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="text">
     <string>Valor de Venda:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_valor_compra">
    <property name="geometry">
     <rect>
      <x>159</x>
      <y>80</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="text">
     <string>Valor de compra:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_valor_venda">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>80</y>
      <width>85</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
    <property name="inputMask">
     <string>R$ xxxxxxxxx; </string>
    </property>
    <property name="text">
     <string>R$ </string>
    </property>
    <property name="maxLength">
     <number>12</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_valor_compra">
    <property name="geometry">
     <rect>
      <x>254</x>
      <y>80</y>
      <width>85</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
    <property name="inputMask">
     <string>R$ xxxxxxxxx; </string>
    </property>
    <property name="text">
     <string>R$ </string>
    </property>
    <property name="maxLength">
     <number>12</number>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="imagens.qrc"/>
 </resources>
 <connections/>
</ui>
